docs/source/user/installing.rst

```diff
@@ -273,33 +273,37 @@ well but were not tested. The use of ``x`` in a version number indicates all
-|        |              |                           |                            |                              | (9.x for aarch64) |                             |
-+--------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
-| 0.50.x | 2020-06-10   | 3.6.x <= version < 3.9    | 1.15 <= version < 1.19     | 0.33.x                       | 9.x               | 2019.5 <= version < 2020.0  |
-+--------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
-| 0.49.x | 2020-04-16   | 3.6.x <= version < 3.9    | 1.15 <= version < 1.18     | 0.31.x <= version < 0.33.x   | 9.x               | 2019.5 <= version < 2020.0  |
-+--------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
-| 0.48.x | 2020-01-27   | 3.6.x <= version < 3.9    | 1.15 <= version < 1.18     | 0.31.x                       | 8.x               | 2018.0.5 <= version < ?     |
-|        |              |                           |                            |                              | (7.x for ppc64le) |                             |
-+--------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
-| 0.47.x | 2020-01-02   | 3.5.x <= version < 3.9;   | 1.15 <= version < 1.18     | 0.30.x                       | 8.x               | 2018.0.5 <= version < ?     |
-|        |              | version == 2.7.x          |                            |                              | (7.x for ppc64le) |                             |
-+--------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
++----------++--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
+| Numba     | Release date | Python                    | NumPy                      | llvmlite                     | LLVM              | TBB                         |
++===========+==============+===========================+============================+==============================+===================+=============================+
+| 0.57.x    | TBC          | 3.8.x <= version < 3.12   | 1.19 <= version < 1.24     | 0.40.x                       | 11.x              | 2021.x                      |
++-----------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
+| 0.56.x    | TBC          | 3.7.x <= version < 3.11   | 1.18 <= version < 1.23     | 0.39.x                       | 11.x              | 2021.x                      |
```

Comment:
Need to change TBC when we get the final release date so as to ensure the stable docs reflect this information.

Reply:
Done in https://github.com/numba/numba/pull/8198/.